translate/rus.json

```diff
@@ -352,7 +353,7 @@
 "shortcut 1":"ярлык 1",
 "shortcut 2":"ярлык 2",
 "show delay":"задержка показа",
-"show ply":"показать слой",
+"show ply":"точка показа ов",
```

Comment:
This one is entirely wrong. The last translation said to me something like 'show the layer', while new one asks the question 'where should I show you the PV, the exact point of it?'. Did you get it all? I hope so.